hathor/p2p/manager.py

```diff
@@ -560,3 +573,24 @@ def drop_connection_by_peer_id(self, peer_id: str) -> None:
         protocol = self.connected_peers.get(peer_id)
         if protocol:
             self.drop_connection(protocol)
+
+    def sync_update(self) -> None:
+        current_enabled: Set[str] = set(
+            conn.peer.id
+            for conn in self.connected_peers.values()
+            if conn.is_sync_enabled()
+        )
+        k = settings.MAX_ENABLED_SYNC
+        selected_peers: Set[str] = set(self.rng.choices(self.connected_peers.keys(), k=k))
```

Comment:
I ran some tests with python `Random` (I know we use a modified class) but can this `choices` select the same peer more than once instead of choosing other peers?

Check this example:

```Python
In [5]: rng = Random()

In [6]: rng.choices([1,2,3], k=4)
Out[6]: [2, 1, 2, 2]
```

It selected the number `2` three times and number `3` wasn't selected. Not sure if something is different in my example

Reply:
Great catch! I'll change the sample method.

Reply:
Fixed!

Reply:
Fixed! Thanks!